medusa/providers/torrent/html/torrentbytes.py

```diff
@@ -139,10 +139,11 @@ def parse(self, data, mode):
                     if not all([title, download_url]):
                         continue
 
-                    if self.freeleech:
-                        # Free leech torrents are marked with green [F L] in the title (i.e. <font color=green>[F&nbsp;L]</font>)
-                        freeleech = cells[labels.index('Name')].find('font', color='green')
-                        if not freeleech or freeleech.get_text(strip=True) != '[F\xa0L]':
+                    # Free leech torrents are marked with green [F L] in the title (i.e. <font color=green>[F&nbsp;L]</font>)
+                    freeleech = cells[labels.index('Name')].find('font', color='green')
+                    if freeleech:
+                        title = title[:-5] # Remove trailing "[F L]"                   
+                        if self.freeleech and freeleech.get_text(strip=True) != '[F\xa0L]':
```

Comment:
@medariox why this other check about freelecher and why '[F\xa0L]' ? if '[F\xa0L]' the [:-5] wont work